app/javascript/components/community-solutions/StarButton.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useState } from 'react'
 import { useMutation } from 'react-query'
-import { FormButton, Icon } from '../common'
 import { sendRequest, typecheck } from '@/utils'
-import { ErrorBoundary, ErrorMessage } from '../ErrorBoundary'
+import { Icon } from '@/components/common'
```

Comment:
You could maybe split this into a separate PR? It seems to not be related to the removing of the `FormButton`?

Reply:
OK!